packages/components/src/components/chat-search/StatusBar.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div :class="['status-bar', `status-bar--${level}`]" data-cy="status-bar">
+    <div class="status-bar__header" data-cy="status-bar-header" @click="toggleShowBody">
+      <span class="status-bar__header__title">
+        <v-warning-icon class="warning-icon" v-if="level === 'warning' || level === 'error'" />
+        <v-success-icon class="success-icon" v-if="level === 'success'" />
+        <slot name="header" />
+      </span>
+      <v-chevron-down-icon :class="['toggle-icon', showBody ? 'down' : 'up']" />
```

Comment:
I feel like the collapsed chevron is reversed. I think it should point down or right.

Reply:
Good point, I fixed this